### ISSUE TITLE
Lots of error messages when starting HA

### ISSUE THREAD

**Issue by DeviousPenguin** [custom-components/hacs#141]
## Version of HACS

 HACS version: 0.8.1 

## Describe the bug

Lots and lots of errors relased to HACS when starting up HA. So far I've only copied over the hacs folder to my custom components folder, and changes my config file.

However HACS seems to be working and I can browse the addon store.

## Debug log

```text

2019-06-19 22:41:36 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.brewdog - manifest.json is missing.
2019-06-19 22:41:36 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.brewdog - Could not register.
2019-06-19 22:41:39 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/hassbian_config - Repository structure does not meet the requirements
2019-06-19 22:41:39 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/hassbian_config - Could not register.
2019-06-19 22:41:41 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.weatheralerts - manifest.json is missing.
2019-06-19 22:41:41 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.weatheralerts - Could not register.
2019-06-19 22:41:42 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.wienerlinien - manifest.json is missing.
2019-06-19 22:41:42 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.wienerlinien - Could not register.
2019-06-19 22:41:44 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/binary_sensor.hadockermon - manifest.json is missing.
2019-06-19 22:41:44 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/binary_sensor.hadockermon - Could not register.
2019-06-19 22:41:52 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sickchill - manifest.json is missing.
2019-06-19 22:41:52 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sickchill - Could not register.
2019-06-19 22:41:53 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/usps_mail - manifest.json is missing.
2019-06-19 22:41:53 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/usps_mail - Could not register.
2019-06-19 22:41:55 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/camera.unsplash - manifest.json is missing.
2019-06-19 22:41:55 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/camera.unsplash - Could not register.
2019-06-19 22:41:56 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/camera.combined - manifest.json is missing.
2019-06-19 22:41:56 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/camera.combined - Could not register.
2019-06-19 22:41:58 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.custom_aftership - manifest.json is missing.
2019-06-19 22:41:58 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.custom_aftership - Could not register.
2019-06-19 22:42:02 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/camera.multisource - manifest.json is missing.
2019-06-19 22:42:02 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/camera.multisource - Could not register.
2019-06-19 22:42:08 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/climate.e_thermostaat - manifest.json is missing.
2019-06-19 22:42:08 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/climate.e_thermostaat - Could not register.
2019-06-19 22:42:10 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.geoip - manifest.json is missing.
2019-06-19 22:42:10 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.geoip - Could not register.
2019-06-19 22:42:24 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.ctabustracker - manifest.json is missing.
2019-06-19 22:42:24 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.ctabustracker - Could not register.
2019-06-19 22:42:30 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/config_check - manifest.json is missing.
2019-06-19 22:42:30 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/config_check - Could not register.
2019-06-19 22:42:37 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/complimentr - manifest.json is missing.
2019-06-19 22:42:37 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/complimentr - Could not register.
2019-06-19 22:42:42 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.airthings_wave - manifest.json is missing.
2019-06-19 22:42:42 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/sensor.airthings_wave - Could not register.
2019-06-19 22:42:47 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/google_keep - No commits
2019-06-19 22:42:47 ERROR (MainThread) [custom_components.hacs.hacs] custom-components/google_keep - Could not register.
2019-06-19 22:43:36 ERROR (MainThread) [custom_components.hacs.hacs] custom-cards/spotify-card - No acceptable js files found
2019-06-19 22:43:36 ERROR (MainThread) [custom_components.hacs.hacs] custom-cards/spotify-card - Could not register.
2019-06-19 22:43:50 ERROR (MainThread) [custom_components.hacs.hacs] custom-cards/playback-card - No acceptable js files found
2019-06-19 22:43:50 ERROR (MainThread) [custom_components.hacs.hacs] custom-cards/playback-card - Could not register.
2019-06-19 22:44:08 ERROR (MainThread) [custom_components.hacs.hacs] custom-cards/muuri-grid - No acceptable js files found
2019-06-19 22:44:08 ERROR (MainThread) [custom_components.hacs.hacs] custom-cards/muuri-grid - Could not register.
```

**Comment by ludeeus** [comment]
This will be fixed in 0.9.0

**Comment by ludeeus** [comment]
Corrected in [0.9.0](https://github.com/custom-components/hacs/releases/0.9.0) 🎉
